backend/grant/email/send.py

```diff
@@ -374,8 +373,14 @@ def send_email(to, type, email_args):
         mail.add_content(Content('text/plain', email['text']))
         mail.add_content(Content('text/html', email['html']))
 
-        res = sg.client.mail.send.post(request_body=mail.get())
-        print('Just sent an email to %s of type %s, response code: %s' % (to, type, res.status_code))
+        if E2E_TESTING:
```

Comment:
Hm, would be nice to not have to alter biz logic for e2e. Not sure how I feel about this.

Reply:
What would be a better way? Is it possible to mock somehow?

Reply:
Monkey patching could be a good option.

Reply:
What about an `@e2e` decorator that uses the wrapped function name to provide alternative logic? It would be a nice mix of explicitly marking where e2e customization exists and localizing e2e logic to one place (away from business logic).

Reply:
I'm kinda into this current approach, since it's tucked away in a non-critical (i.e. not a database altering operation) utility function. If these were sprinkled all over the views, it'd rustle my jimmies a little more.